composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "3e076f672514ba76ba4f5c6a91c9c9f5",
+    "content-hash": "4ad8d7921c1ad8f1df1d3fd7510bfe2e",
```

Comment:
Why so many changes to the lock file? Was it produced after `composer update`?

Reply:
Yes, after running `composer update`